keyboards/kprepublic/bm60hsrgb/rev2/keymaps/tcdejw/config.h

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2023 TCDejw
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+// disable not used functions for more space
+#define NO_ACTION_MACRO
+#define NO_ACTION_FUNCTION
+#define NO_ACTION_ONESHOT		//Save 244 bytes (-244).
+#define NO_RESET				//Save 40 bytes (-40).
```

Comment:
Saved bytes is highly compiler version dependant.  As such, they shouldn't be listed, as it can drastically change. 
```suggestion
#define NO_ACTION_ONESHOT
#define NO_RESET
```

Also, `NO_RESET` disables the bootloader/reset keycodes.  Not sure if that is your desired intent.

Reply:
Hi, thank you for your change, I appreciate it.
> Also, NO_RESET disables the bootloader/reset keycodes. Not sure if that is your desired intent.
It is, since BM60 (poker and non-poker one) have physical reset button to restart directly to bootloader. Reset keycodes are not necessary in my case.